docs/examples/balltree-kdtree-performance.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>Better to have exact same coords for both queries</p><p>3.0 3.0 spherical and 0.0 0.0 1.0 for cartesian are different.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>